src/makeInfinityAdder.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  const next = (...args) => {
+    return (x) => {
+      if (x === undefined) {
+        return args.reduce((sum, a) => sum + a, 0);
```

Comment:
Can you try to avoid reduce? I mean: this task can be solved without using a loop